src/main/java/gov/usds/case_issues/config/OAuthMappingConfig.java

```diff
@@ -83,19 +86,24 @@ public WebSecurityPlugin oauthConfigurer() {
 	}
 
 	protected static OAuth2UserService<OAuth2UserRequest, OAuth2User> createDelegatingUserService(
-			final OAuth2UserService<OAuth2UserRequest, OAuth2User> delegate, List<String> inputPath) {
+			final OAuth2UserService<OAuth2UserRequest, OAuth2User> delegate, List<String> inputPath,
+			UserService u) {
```

Comment:
By convention, the closed-over service object parameter should be declared `final`, though as you've noticed, it probably doesn't have to be (it used to be required, but it's still IMO a useful flag for "this is going to get used in a lambda, be careful with it").